omnibus/resources/chef-workstation/msi/assets/create-chef-dir.ps1

```diff
@@ -0,0 +1,6 @@
+$chef_path = "$HOME\.chef"
+
+If(!(test-path $chef_path))
+{
+  New-Item -ItemType Directory -Force -Path $chef_path
```

Comment:
Why do you force this action? `-Force`

Reply:
to prevent it from blowing up if that's already there

Reply:
which actually shouldn't be necessary anymore

Reply:
Fixed